src/test/java/com/artipie/rpm/RpmTest.java

```diff
@@ -53,16 +49,19 @@
     private static final String REPOMD = "repodata/repomd.xml";
 
     @Test
-    @Disabled
-    void doesntBrakeMetadataWhenInvalidPackageSent(@TempDir final Path tmp)
+    void doesntBrakeMetadataWhenInvalidPackageSent()
```

Comment:
@olegmoz this test does not seem accurate enough: except repomd rpm has other metadata files, that should not be changed to. So I think we need todo to extend this test.